tools/harness-simulation/harness/Thread_Harness/Sniffer/SimSniffer.py

```diff
@@ -259,8 +260,18 @@ def startSniffer(self, channelToCapture, captureFileLocation, includeEthernet=Fa
         if response.status != sniffer_pb2.OK:
             raise RuntimeError('startSniffer error: %s' % sniffer_pb2.Status.Name(response.status))
 
+        self._thread = threading.Thread(target=self._file_sync_main_loop)
+        self._thread.start()
+
         self.is_active = True
 
+    @watched
+    def _file_sync_main_loop(self):
+        with open(self._local_pcapng_location, 'w+b') as f:
```

Comment:
```suggestion
        with open(self._local_pcapng_location, 'wb') as f:
```
why is `+` necessary?

Reply:
`+` is not necessary. I misunderstood the usage of `w+`. Copying an unclosed file opened in mode `w` will not fail.